frontend/package.json

```diff
@@ -11,33 +11,81 @@
     "url": "https://openframeworks.cc"
   },
   "engines": {
-    "node": ">=12.20.55"
+    "node": ">=16.0.0"
   },
   "scripts": {
     "start": "electron .",
-    "build": "electron-packager . $npm_package_productName --icon=static/icon/of.icns --out=dist --ignore='dist' --ignore='readme.md' --prune --all --overwrite --electron-version=25.3.0",
-    "build:osx": "echo 'DEPRECATED: build:osx is deprecated. please use build:macos instead' && npm run build:macos",
-    "build:macos": "electron-packager . $npm_package_productName --icon=static/icon/of.icns --out=dist --ignore='dist' --ignore='readme.md' --prune --overwrite --electron-version=25.3.0 --platform=darwin --arch=universal --osxUniversal.x64ArchFiles=Contents/Resources/app/app/projectGenerator",
-    "build:macos-m1": "electron-packager . $npm_package_productName --icon=static/icon/of.icns --out=dist --ignore='dist' --ignore='readme.md' --prune --overwrite --electron-version=25.3.0 --platform=darwin --arch=arm64",
-    "build:linux32": "electron-packager . $npm_package_productName --icon=static/icon/of.icns --out=dist --ignore='dist' --ignore='readme.md' --prune --overwrite --electron-version=25.3.0 --platform=linux --arch=ia32",
-    "build:linux64": "electron-packager . $npm_package_productName --icon=static/icon/of.icns --out=dist --ignore='dist' --ignore='readme.md' --prune --overwrite --electron-version=25.3.0 --platform=linux --arch=x64",
-    "build:vs": "electron-packager . --icon=static/icon/of.ico --out=dist --ignore='dist' --ignore='readme.md' --prune --overwrite --electron-version=25.3.0 --platform=win32 --arch=ia32"
+    "start:prod": "electron .",
+    "dist": "electron-builder",
+    "dist:mac:x64": " electron-builder --mac --x64 --publish never",
+    "dist:mac:arm64": " electron-builder --mac --arm64 --publish never",
+    "dist:mac:universal": " electron-builder --mac --universal --publish never",
```

Comment:
does this in tern call build:mac ? 
is that where the: 

```
   "mergeASARs": false,
       "x64ArchFiles": "Contents/Resources/app/app/projectGenerator"
```
are getting included from?

Reply:
Yeah all the ```mac:x``` calls use the mac { }.
The only one being used though in ci currently is ```dist:mac:universal```

Reply:
okay - awesome. 
looks good to me then!! 👍